web/utils/storybook-utils.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
This fn is useful when writing stories for components, as it allows fields to be easily ommitted from the controls panel in Storybook.